Std/Tactic/Lint/Frontend.lean

```diff
@@ -134,23 +143,24 @@ def groupedByFilename (results : HashMap Name MessageData) : CoreM MessageData :
   let grouped' := grouped.toArray.qsort fun (a, _) (b, _) => toString a < toString b
   (MessageData.joinSep · (Format.line ++ Format.line)) <$>
     grouped'.toList.mapM fun (mod, msgs) => do
-      pure m!"-- {mod}\n{← printWarnings msgs}"
+      let fp := modToFilePath ⟨"."⟩ mod "lean"
```

Comment:
This is definitely wrong when the executable isn't being run from the project root, or when another package is being linted. It needs to use `getPackageDir`